### PR TITLE
Limit API access

### PR DESCRIPTION
**Pull request by Ecnama**
## Description

Forbids outside access to the API by binding Fastapi to the docker bridge IP.

## Checklist

- [x] I have tested the changes locally and they work as expected.
- [x] I have tested the responsiveness of the changes and they work as expected.
- [x] I have assigned the pull request to the appropriate reviewer(s).
- [x] I have added labels to the pull request, if necessary.

### CODE REVIEW

File: netcontrol/fastapi_init.py
Comment (pixup1) [inline netcontrol/fastapi_init.py:1]: Y'a des fois où bash c'est mieux quand même xD

Comment (Ecnama) [reply netcontrol/fastapi_init.py:1]: Pas faux, j'aime pas trop bash, mais je peux le refaire, ce sera plus propre

**Review by pixup1 — CHANGES_REQUESTED** [review]
Use an nftable at launch to limit interaction to loaclhost addresses only. Scrap the bash script.

**Review by pixup1 — APPROVED** [review]
✨✨ 🄻🅶🅃🅼 ✨✨

**Pull request merged**